build.gradle.kts

```diff
@@ -174,17 +174,10 @@ tasks {
         sinceBuild.set(properties("pluginSinceBuild"))
         untilBuild.set(properties("pluginUntilBuild"))
 
-        pluginDescription.set(
-            File("$projectDir/README.md").readText().lines().run {
-                val start = "<!-- Plugin description start -->"
-                val end = "<!-- Plugin description end -->"
-
-                if (!containsAll(listOf(start, end))) {
-                    throw GradleException("Plugin description section not found in README.md file:\n$start ... $end")
-                }
-                subList(indexOf(start) + 1, indexOf(end))
-            }.joinToString("\n").run { markdownToHTML(this) }
-        )
+        val content = File("$projectDir/README.md").readText()
+        val startIndex = content.indexOf("# JetBrains plugins")
```

Comment:
A fix for this has already been delivered in https://github.com/snyk/snyk-intellij-plugin/commit/a1aa7592c44a0b1220829f4344afe39e4ca5525c, might need to make some changes here